teamB/junhyungcho/services/userService.js

```diff
@@ -0,0 +1,42 @@
+//실제 로직이 돌아가는 파일
+const { userDao } = require('../models/')
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
```

Comment:
require 순서는 다음과 같이 하면 좋을 것 같습니다.
```suggestion
// 3rd-part package
const bcrypt = require('bcrypt');
const jwt = require('jsonwebtoken');

//custom package
const { userDao } = require('../models/')
```